src/app/design/styles/typography/index.html

```diff
@@ -4,7 +4,7 @@
 
   <stache-page-summary>
     <p>
-      SKY UX supports a set of semantically defined classes for text to ensure consistent usage and to maintain visual hierarchy of type. Components generally incorporate text classes as necessary, but you may find cases where you need to apply classes directly. The SKY UX font system is based on the Blackbaud Sans typeface. Styling values in the table below are for reference only; you should apply styling through the use of the classes.
+      SKY UX supports a set of semantically defined classes for text to ensure consistent usage and to maintain visual hierarchy of type. Components generally incorporate text classes as necessary, but you may find cases where you need to apply classes directly. The SKY UX font system is based on the Blackbaud Sans typeface, which you can use in web applications by linking directly to the <a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans.woff">Blackbaud CDN</a>. Styling values in the table below are for reference only; you should apply styling through the use of the classes.
```

Comment:
Doing a bit of thinking out loud here, but how about something like the following:

- Move your new sentence from the middle of this paragraph in to its own paragraph (so it has gains a little more attention).
- Change it to something like "The SKY UX font system is based on the Blackbaud Sans typeface, which you can use in web and desktop applications.  Learn more (where the learn more points to the [resources page](https://developer.blackbaud.com/skyux/design/resources)).
- On the [resources page](https://developer.blackbaud.com/skyux/design/resources), add a new paragraph under the fonts section with something like "The Blackbaud Sans typeface is available on the Blackbaud CDN..."

Reply:
+1 on all of Bobby's suggestions. I think that's a great plan. ... A few small additional suggestions though: In the new standalon pagraph, I'd split the content into 2 sentences instead of using a "which" clause to make a long sentence. I'd also add "applications" after "web" because I think that flows better. And I'd flesh out the last sentence with the cross reference: "The SKY UX font system is based on the Blackbaud Sans typeface. To use these fonts in web applications and desktop applications, you link directly to the Blackbaud CDN. For more information, see our [design resources](https://developer.blackbaud.com/skyux/design/resources)."